chart/elastalert2/values.yaml

```diff
@@ -61,6 +61,10 @@ deploymentAnnotations: {}
 # Annotations to be added to pods
 podAnnotations: {}
 
+quickwit:
+  # enable quickwit interoperability
+  enable: false
```

Comment:
Is this used somewhere? The chart readme will need to be updated with this setting if this is intended to remain. The readme should mention how this will work in tandem with the `qw_enable` setting.

Reply:
Yes. It's use in the [`chart/elastalert2/templates/config.yaml` file](https://github.com/idrissneumann/elastalert2/blob/f2e3e31bb721d840ee37d849638b7a9c8a4eb499/chart/elastalert2/templates/config.yaml#L33):

```helm
{{- if .Values.quickwit.enable }}
    qw_enable: "True"
{{- end }}
```

And the README of the helmchart also mention `quickwit.enable` [here](https://github.com/idrissneumann/elastalert2/blob/f2e3e31bb721d840ee37d849638b7a9c8a4eb499/chart/elastalert2/README.md?plain=1#L64).

I thought it's more consistent with the rest of the helm configuration (we can see that it's the same way for elasticsearch) and more "helm friendly" (There's often a block for configuring a dependency with an `enable` flag).